components/FileUpload/FileUpload.jsx

```diff
@@ -6,12 +6,13 @@ import DeleteIcon from 'components/Icons/DeleteIcon';
 
 const FileUpload = ({
   label,
+  sublabel,
+  hint,
   name,
   fileList = [],
   onFileUploaded,
   onFileDelete,
-  uploadPrefix = 'test',
-  isSingleFile
+  uploadPrefix = 'test'
```

Comment:
Do we still want this?

Reply:
it's `test` as a fallback, if it's not provided when invoking the function

Reply:
Ah okay no worries :) lgtm